src/olympia/devhub/tests/test_tasks.py

```diff
@@ -1320,8 +1347,8 @@ def test_validation_finishes_if_containing_invalid_filename(self):
         upload.refresh_from_db()
 
         # https://github.com/mozilla/addons-server/issues/8208
-        # causes this to be 2 (and invalid) instead of 0 (and valid).
+        # causes this to be 1 (and invalid) instead of 0 (and valid).
         # The invalid filename error is caught and raised outside of this
         # validation task.
-        assert upload.processed_validation['errors'] == 2
+        assert upload.processed_validation['errors'] == 1
```

Comment:
? I don't see any validation changes in this patch?

Reply:
The extra `target_apps` condition removes one of the bogus errors.